src/mame/drivers/srmp6.cpp

```diff
@@ -704,7 +704,7 @@ void srmp6_state::srmp6(machine_config &config)
 	SPEAKER(config, "lspeaker").front_left();
 	SPEAKER(config, "rspeaker").front_right();
 
-	nile_device &nile(NILE(config, "nile", 0));
+	nile_sound_device &nile(NILE_SOUND(config, "nile", XTAL(42'954'545) / 1113)); // 56MHz / 1451 or 42.9545MHz / 1113?
```

Comment:
Where did you get the numbers from?  The divider seems like a weird number.

Reply:
1113 has prime factors of 3, 7 and 53.  1451 is a prime number.  Although possible, it’s fairly unusual to see dividers with large prime factors like this.